bindata/network/ovn-kubernetes/managed/alert-rules.yaml

```diff
@@ -316,6 +316,23 @@ spec:
       for: 15m
       labels:
         severity: warning
+    # OVN northbound and southbound database performance alerts
+    - alert: OVNKubernetesNorthboundDatabaseCPUUsageHigh
+      annotations:
+        summary: OVN northbound database {{ $labels.instance }} is greater than {{ $value }} percent CPU usage for a period of time.
+        description: High OVN northbound CPU usage indicates high load on the networking control plane.
+      expr: (sum(rate(container_cpu_usage_seconds_total{container="nbdb"}[5m])) BY (instance, name) * 100) > 80
+      for: 15m
+      labels:
+        severity: info
+    - alert: OVNKubernetesSouthboundDatabaseCPUUsageHigh
+      annotations:
+        summary: OVN southbound database {{ $labels.instance }} is greater than {{ $value }} percent CPU usage for a period of time.
+        description: High OVN southbound CPU usage indicates high load on the networking control plane.
+      expr: (sum(rate(container_cpu_usage_seconds_total{container="sbdb"}[5m])) BY (instance, name) * 100) > 80
```

Comment:
80% seems safe, in my experience anything over 70% is a warning sign SBDB is in trouble